runtimes/common/src/dip/README.md

```diff
@@ -0,0 +1,16 @@
+# KILT Decentralized Identity Provider (DIP) provider specification
+
+Specification of the format of a DIP identity commitment and the expected format of a DIP identity proof for cross-chain transactions using KILT identities.
+
+## V0
+
+The V0 of the KILT DIP Provider specification defines the following components:
+
+* **Identity details**: What are the pieces of a KILT identity that can be used for cross-chain transactions. V0 defines them to include the following information:
+  * All `DidKey`s stored under the subject's DID Document. For more details about how these keys are defined, please check the [KILT DID pallet](../../../../pallets/did).
```

Comment:
And same here, we should probably replace all these with hard URLs when we have them.

Reply:
Will keep this (and the others) open as a reminder.